src/stylesheets/elements/_buttons.scss

```diff
@@ -45,12 +44,6 @@ button,
     text-decoration: none;
   }
 
-  &:focus,
-  &.usa-button-focus {
```

Comment:
The removal of some of these classes makes it so the focus state isn't being show on the fractal example like it should (example of how it should look below)

![screen shot 2017-08-21 at 2 13 14 pm](https://user-images.githubusercontent.com/776987/29534253-ed49ff20-867a-11e7-9922-fc97a1da5fb3.png)

Doesn't have to be done in the same fashion as the past but some `njk` files may need to be adjusted otherwise.